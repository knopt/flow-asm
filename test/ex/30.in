1
29
-4.399441002055678
-8.880576820172772 
5.78480448911121 
1.7968153886851486 
6.732802081933741 
-4.226692090481466 
8.03427447113878 
-9.27841320769988 
-1.68489560799844 
-1.187021037533551 
-0.9858663004663981 
8.999979340629807 
7.4411440678476986 
2.3302297743070515 
6.731204229078514 
4.639852470161877 
-2.860187126164522 
-1.5567500152632245 
-8.689867163476313 
-3.220548221271418 
1.7191681875574751 
4.660400867051788 
2.8732358152535937 
2.411843487346548 
-2.234432355963687 
-7.48810840713184 
1.4688806604311644 
5.847158248960348 
-6.082115967934206 
8.082174257143052 

6.961485438784244 3.842641334227883 -6.03014183197293 4.698076070352721 3.7308853761240677 1.7481468282718193 9.319324906380334 -7.868717901051026 -6.414784737568738 9.159480946083182 -0.3548962645605158 0.19457959721593632 -8.878034501152099 4.084615848580331 9.048431875133101 -1.080018233179878 -1.55869675533555 -6.947030102593872 5.680695217994568 1.664792965233497 3.4147538061442457 8.633011819348265 5.90648266040769 8.132174044106272 -2.1298390171506787 7.2410765570838755 9.232776067059326 6.985500587891494 -2.5065639007195806 
