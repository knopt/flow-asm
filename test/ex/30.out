-199.07132254876137 
143.1960601781323 
-24.793868849930938 
154.9722635950983 
-206.94257401449582 
179.0762720713118 
-255.38401718666174 
29.133777861445715 
6.994524424724419 
-67.55860239068477 
138.98346398314064 
161.04262181885434 
11.470682749870315 
105.41473837252923 
36.6467349719011 
-107.53940834433858 
57.184803156132766 
-166.39665490085835 
-45.14942161996382 
-14.137533310442224 
26.644591194566665 
-44.02618477311278 
-47.08738708873628 
-81.44511176608786 
-227.10117258940343 
-22.106228504889444 
51.555913140109325 
-261.4768156688637 
121.8064133564769 
